### PR TITLE
Seq tests

### PR DESCRIPTION
**Pull request by ts-klassen**
## Summary by CodeRabbit

- **New Features**
  - Added support for the `seq` command, enabling generation of numeric sequences with formatting, custom separators, and zero-padding options.
- **Bug Fixes**
  - Tests are now generated for all commands, not just recognized built-in commands.
- **Tests**
  - Introduced extensive automated tests for the new `seq` command, covering various argument combinations, formatting, error handling, and edge cases.
  - Added a test for the `cat -n` command.

### CODE REVIEW

File: tests/auto_generated/seq_1_10.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Auto-generated from seq_1_10.txt. Do not edit.
+const { expect } = require('chai');
+const klsh = require('../../dist/klsh.js');
+
+describe('auto-generated seq_1_10', function() {
+  it("bash: seq 1 10", function() {
+    const stdin = "\n";
```
[the remark this answers is not in the document]
Comment (ts-klassen) [reply tests/auto_generated/seq_1_10.test.js:7]: Please don''t review auto generated files.
